code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -51,6 +51,7 @@
 	flags = FPRINT
 	origin_tech = Tc_MAGNETS + "=3;" + Tc_SYNDICATE + "=4"
 	attack_verb = list("attacks", "slashes", "stabs", "slices", "tears", "rips", "dices", "cuts")
+	clumsy_check = 1
```

Comment:
var/clumsy_check exists at some level on weapons, I'll probably make a seperate PR to set it as a weapon variable and to turn all the clumsy checks into a proc as opposed to copypaste.